challenge2/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorign>
+    <link href="https://fonts.googleapis.com/css2?family=Manrope&family=Ubuntu:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+
+
+    <main>
+        <section class="main-section">
+            <div class="image-container">
+                <img src="./assets/imgs/principal.svg" alt="Example Image">
+            </div>
+            <div class="form-container">
+                <div class="title">
+                    <h1>Sign Up</h1>
+                    <p>If you already have an account register</p>
+                    <p>You can <a class="link">Login here!</a></p>
+                </div>
+
+                <div class="info">
+                    <form>
```

Comment:
I think this `div` that wraps the `form` is not needed. When we have a div with only one child, we can usually remove it by moving its styles to the child (the form in this case)